store/sqlstore/post_store.go

```diff
@@ -625,28 +646,29 @@ func (s *SqlPostStore) getPostsAround(channelId string, postId string, limit int
 				rootIds = append(rootIds, post.RootId)
 			}
 		}
+		rootQuery := s.getQueryBuilder().Select("p.*")
+		if options.SkipFetchThreads {
+			rootQuery = rootQuery.Column(sq.Alias(replyCountSubQuery, "ReplyCount"))
+		}
+		rootQuery = rootQuery.From("Posts p").
+			Where(sq.And{
+				sq.Or{
+					sq.Eq{"RootId": rootIds},
```

Comment:
Are we getting here all the "siblings"? that posts that have a `RootId` equal to others `RootId`? Shouldn't it be getting only the "roots" using only the `Id` clause (if we are "SkipingFetchThreads")?

Reply:
we are getting the posts + their roots. the replyCount is written into the root post